### PR TITLE
Create mikko.json

### PR DESCRIPTION
**Pull request by mikkoxs**

### CODE REVIEW

File: domains/mikko.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Mikko's Developer Page",
+  "repo": "https://github.com/mikkortx/cv",
+  "owner": {
+    "username": "mikkortx",
+    "email": "mikkoahh@gmail.com",
+    "twitter": "mikkortx"
+  },
+  "record": {
+    "CNAME": "https://mikkortx.github.io/cv/"
```
Comment (MaskDuck) [inline domains/mikko.json:10]: cname cant have slash

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/mikko.json (hunk 1 above)
Comment (MaskDuck) [reply domains/mikko.json:10]: want a url redirection? use an url record
want a masked website? change cname value to `mikkortx.github.io`.

Comment (mikkortx) [reply domains/mikko.json:10]: sure please

Comment (MaskDuck) [reply domains/mikko.json:10]: have you resolve it

**Review by MaskDuck — APPROVED** [review]
looks good now

**Comment by MaskDuck** [comment]
if ci doesnt complain then ill merge

**Pull request merged**